src/testing/test-helper.js

```diff
@@ -289,10 +289,6 @@ function checkAuthorizations(expectedAuthorizations, actualAuthorizations, autho
     expectedAuthorizations = [ expectedAuthorizations ];
   }
 
-  if (!(actualAuthorizations instanceof Array)) {
-    actualAuthorizations = [ actualAuthorizations ];
-  }
```

Comment:
This step is no longer necessary since all `requireAccess`, `requireRole` and `requireUser` calls are now passed an array.